pkg/cmd/taskrun/taskrun_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package taskrun
+
+import (
+	"testing"
+
+	"github.com/tektoncd/cli/pkg/test"
+)
+
+func TestPipelines_invalid(t *testing.T) {
```

Comment:
same :)

Reply:
:man_facepalming: :joy:

Reply:
oops sorry i should have done a better review 🙇

Reply:
not a big deal, its just a name and was a nit .. importantly we have a test :) 
🤗

Reply:
it has some usefulness still I think when user want to only run a specific test with go run, i.e: `-run TestBlahRun`